docs/src/understand/tedge-agent.md

```diff
@@ -0,0 +1,60 @@
+---
+title: The Agent
+tags: [Concept, MQTT]
+sidebar_position: 5
+---
+
+# Thin-edge Agent
+
+Thin-edge agent is a set of services that implement the device management operations on a device:
+- software management, checking installed packages and updating these packages
+- configuration management, checking and updating configuration files
+- firmware management, updating the firmware of the device
+- device restart
+- remote access to a console on the device from the cloud
+- log file management, retrieving excerpts from log files
+
+:::note
+In the current version of thin-edge, the agent features are not implemented by a single executables,
+but by a set of executables:
+
+- `tedge-agent`
+- `c8y-configuration-plugin`
+- `c8y-firmware-plugin`
+- `c8y-log-plugin`
+- `c8y-remote-access-plugin`
+
+The short-term plan is to re-organize these plugins to move the Cumulocity aspects in the Cumulocity
+and to group the management operations in a single executable. 
+:::
+
+Thin-edge agent acts as a device connector:
+- listening to operation requests published on the MQTT bus
+- delegating the actual operations to the operating system
+- reporting progress of the requests
+
+## Operation MQTT topics
+
+Operation requests are published by the requesters on operation specific topics:
+
+- `tedge/commands/req/${operation-type}/${operation-action}`
+- `${operation-type}/${operation-action}` is the well-known name of the operation request,
+  such as `software/update` or `control/restart`
+- the payloads are operation-specific
+
+The corresponding operation responses are published to associated topics:
+
+- `tedge/commands/res/${operation-type}/${operation-action}`.
```

Comment:
Just more involved markdown refactoring to make it more readable.

````suggestion
```text
tedge/commands/req/{operation-type}/{operation-action}
```

Where the combination of `operation-type` and `operation-action` is the well-known name of the operation request, such as:
* `software/update`
* `control/restart`

The payload sent to the topics to trigger the action is also operation specific.

The corresponding operation responses are published to associated topics:

```text
tedge/commands/res/{operation-type}/{operation-action}
```
````

Reply:
Updated